### PR TITLE
Extend vpc module

### PR DESCRIPTION
**Pull request by OpsStrike**
@ggramal

### CODE REVIEW

File: terraform/environments/prd/aws/main.tf [hunk 1]
```diff
@@ -42,13 +42,24 @@ module "vpcs" {
   nat_gws      = local.vpcs["ivanoff-tf"].nat_gws
   subnets      = local.vpcs["ivanoff-tf"].subnets
 }
+
+output "vpc_private_subnet_ids" {
```
Comment (ggramal) [inline terraform/environments/prd/aws/main.tf:46]: why do we need to ouput private  subnets in root module?

Comment (OpsStrike) [reply terraform/environments/prd/aws/main.tf:46]: For that they are avaible to other modules

Comment (OpsStrike) [reply terraform/environments/prd/aws/main.tf:46]: and for see password in output

Comment (ggramal) [reply terraform/environments/prd/aws/main.tf:46]: what password are we talking about if this PR is about VPC

Comment (ggramal) [reply terraform/environments/prd/aws/main.tf:46]: >For that they are avaible to other modules

What modules?

Comment (OpsStrike) [reply terraform/environments/prd/aws/main.tf:46]: > what password are we talking about if this PR is about VPC

thought about one thing wrote about another) 
Output variables allow data created and managed within a module to be accessed from outside that module. export information about private and public subnets created in the VPC module.
or pass it to other modules

**Review by ggramal — DISMISSED** [review]

**Review by CyberCuCuber — APPROVED** [review]

**Pull request merged**